### PR TITLE
feat: added camera icons and fixed formatting errors

### PR DESCRIPTION
**Pull request by bethanybeech**
- [ ] **BREAKING CHANGE?** 

## Description

Added 16px and 12px stroke and fill versions of the camera icon

## Checklist

* [x] :ok_hand: SVG updates are Garden Designer approved (add the
  designer as a reviewer)
* [x] :globe_with_meridians: SVG demo is up-to-date (`yarn start`)
* [ ] :black_medium_small_square: Renders as expected in "dark" mode
* [ ] :white_large_square: Renders as expected @ 2x scale

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ dist
 node_modules
 package-lock.json
 *.log
+.DS_Store
```
Comment (Francois-Esquire) [inline .gitignore:7]:
```suggestion
```

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,4 @@ demo
 dist
 node_modules
 package-lock.json
-*.log
+*.log
```
Comment (jzempel) [inline .gitignore:6]: [nit] can we restore the newline at the end of this file? (ideally, it won't appear in the diff)

**Review by jzempel — CHANGES_REQUESTED** [review]
it would be good to eek out a bit of cost savings on the size by combining `path` elements into one. lmk if you'd like me to make these optimizations.

**Comment by bethanybeech** [comment]
@jzempel if it's not too much trouble for you to make the reformatting changes, I will take you up on your offer! It's probably faster than me trying to figure it out :)

**Review by jzempel — APPROVED** [review]

**Pull request merged**